cmd/minikube/cmd/service_test.go

```diff
@@ -30,10 +30,10 @@ func TestCheckEndpointReady(t *testing.T) {
 
 	endpointNotReady := &kubeApi.Endpoints{
 		Subsets: []kubeApi.EndpointSubset{
-			{Addresses: []kubeApi.EndpointAddress{
```

Comment:
nit: you may want to reword the error text returned in these two tests as they don't seem to match the logic anymore

https://github.com/dlorenc/minikube/blob/53bed6f594b83ca5caf251a7a4ec89c0cd65554b/cmd/minikube/cmd/service_test.go#L40

https://github.com/dlorenc/minikube/blob/53bed6f594b83ca5caf251a7a4ec89c0cd65554b/cmd/minikube/cmd/service_test.go#L53